airbyte-integrations/connectors/source-mongodb-v2/README.md

```diff
@@ -1,5 +1,27 @@
 # MongoDb Source
 
+## Schema
+
+For each collection, if you want to define a schema in order to propagate type to Glue, specify the schema for the collection as a unique string. 
+
+Example:
+
+embedded_movies:{"fields":[{"name":"_id"COMMA"type":"string"}COMMA{"name":"tomatoes"COMMA"type":"object"COMMA"subfields":[{"name":"production"COMMA"type":"string"}]}]}
+
+This means that the collection `embedded_movies` has a schema with a field `_id` of type `string` and a field `tomatoes` of type `object` with a subfield `production` of type `string`.
+
+Valid types are:
```

Comment:
Should be any valid type here, right: 

https://docs.airbyte.com/understanding-airbyte/supported-data-types

?

Reply:
This is using the existing mongo source conversion function thats applied already in the vanilla connector

https://github.com/Encore-Post-Sales/airbyte-magnify/blob/magnify-main/airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/MongoUtil.java#L368